src/ProjectTemplates/Web.ItemTemplates/content/Protobuf/protobuf.proto

```diff
@@ -0,0 +1 @@
+syntax = "proto3";
```

Comment:
Looking at the other item templates we ship, we seem to make it as bare bones as possible, so this is the closest equivalent for proto files. Should we add content for sample service, rpc, or message definitions @shirhatti ?

Reply:
Right now I'm inclined to agree. Especially since you don't know whether it's a service or a message.
I could be convinced otherwise though